R/test.r

```diff
@@ -47,7 +47,7 @@ test <- function(pkg = ".", filter = NULL, ...) {
   # Run tests in a child of the namespace environment, like
   # testthat::test_package
   message("Loading ", pkg$package)
-  ns_env <- load_all(pkg, quiet = TRUE)$env
+  ns_env <- load_all(pkg, quiet = TRUE, export_all = FALSE)$env
```

Comment:
Yeah it does (just checked). I wonder we could come up with an easy test for fully exported namespace and set `export_all` based on that...